lib/chat_api/accounts.ex

```diff
@@ -37,7 +37,7 @@ defmodule ChatApi.Accounts do
 
   """
   def get_account!(id) do
-    Account |> Repo.get!(id) |> Repo.preload(:users)
+    Account |> Repo.get!(id) |> Repo.preload(:users) |> Repo.preload(:widget_config)
```

Comment:
I ~think we can just do `Repo.preload([:users, :widget_config])` rather than calling `preload` twice